charts/kargo/templates/_helpers.tpl

```diff
@@ -55,9 +55,10 @@ Generate the base URL for the API service.
 Generate the base URL for the external webhook server.
 */}}
 {{- define "kargo.externalWebhooksServer.baseURL" -}}
+{{- $apiService := .Values.api -}}
 {{- $webhookService := .Values.externalWebhooksServer -}}
-{{- if and (not $webhookService.ingress.enabled) (not $webhookService.tls.enabled) (not $webhookService.tls.terminatedUpstream) -}}
-{{- include "kargo.api.baseURL" . -}}
+{{- if and (not $webhookService.ingress.enabled) $apiService.enabled $apiService.ingress.enabled -}}
+{{- printf "%s/webhooks" (include "kargo.api.baseURL" .) -}}
```

Comment:
If API Ingress is enabled for the API server and the external webhooks server does not have its own dedicated ingress, the API server ingress will route `/webhook` to the external webhooks server.

We were not accounting for thins in constructing the external webhook server's base URL.